migrations/20161019214312-addingIsRoot.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
```

Comment:
Can we change the name of this file to *-add-is-root-to-items.js?  I prefer the imperative wording, hyphens in the name, and a meaningful description of the update that will occur.